docs/GettingStarted.md

```diff
@@ -50,89 +65,35 @@ To enable the Rails auto instrumentation, create an initializer file in your ``c
-To integrate Rails instrumentation with third-party libraries such as Grape, please check the available settings below.
-
-#### Configure the tracer with initializers
+    Datadog.configure do |c|
+      c.use :rails
+    end
 
-All tracing settings are namespaced under the ``Rails.configuration.datadog_tracer`` hash. To change the default behavior
-of the Datadog tracer, you can override the following defaults:
+To use service names other than the default ones, you might provide custom options as:
 
     # config/initializers/datadog-tracer.rb
 
-    Rails.configuration.datadog_trace = {
-      enabled: true,
-      auto_instrument: false,
-      auto_instrument_redis: false,
-      auto_instrument_grape: false,
-      default_service: 'rails-app',
-      default_controller_service: 'rails-controller',
-      default_cache_service: 'rails-cache',
-      default_database_service: 'postgresql',
-      distributed_tracing_enabled: false,
-      template_base_path: 'views/',
-      tracer: Datadog.tracer,
-      debug: false,
-      trace_agent_hostname: 'localhost',
-      trace_agent_port: 8126,
-      env: nil,
-      tags: {}
-    }
-
-Available settings are:
+    Datadog.configure do |c|
+      c.use :rails, option_name: option_value, ...
+    end
+
+Available options for the `rails` integration are:
 
-* ``enabled``: defines if the ``tracer`` is enabled or not. If set to ``false`` the code could be still instrumented
-  because of other settings, but no spans are sent to the local trace agent.
-* ``auto_instrument``: if set to +true+ the code will be automatically instrumented. You may change this value
-  with a condition, to enable the auto-instrumentation only for particular environments (production, staging, etc...).
-* ``auto_instrument_redis``: if set to ``true`` Redis calls will be traced as such. Calls to Redis cache may be
-  still instrumented but you will not have the detail of low-level Redis calls.
-* ``auto_instrument_grape``: if set to ``true`` and you're using a Grape application, all calls to your endpoints are
-  traced, including filters execution.
-* ``default_service``: set the service name used when tracing application requests. Defaults to ``rails-app``
-* ``default_controller_service``: set the service name used when tracing a Rails action controller. Defaults to ``rails-controller``
-* ``default_cache_service``: set the cache service name used when tracing cache activity. Defaults to ``rails-cache``
-* ``default_database_service``: set the database service name used when tracing database activity. Defaults to the
-  current adapter name, so if you're using PostgreSQL it will be ``postgres``.
-* ``distributed_tracing_enabled``: enable [distributed tracing](#Distributed_Tracing) so that this service trace is
+* ``service_name``: set the service name used when tracing application requests (on the `rack` level). Defaults to ``<app_name>``, where `<app_name>` is automatically inferred from your Rails application namespace;
```

Comment:
I'd suggest adding the expected/allowed types (e.g. String, Symbol, Array etc.  Perhaps giving an example value would be good too.)